lib/geo.ex

```diff
@@ -29,7 +29,7 @@ defmodule Geo do
   """
   @type endian :: :ndr | :xdr
 
-  if Application.get_env(:geo, :impl_to_string, true) do
+  if Application.compile_env(:geo, :impl_to_string, true) do
```

Comment:
If @bryanjos still wants to support Elixir versions < 1.10 (which I don't think is necessary since Elixir 1.10 is now the oldest version receiving security patches), this line can be changed to:

```elixir
if apply(Application, :get_env, [:geo, :impl_to_string, true]) do
```

to [trick the compiler](https://elixirforum.com/t/elixir-v1-14-0-rc-1-released/49581/11) and not get the deprecation warning.